packages/react/src/products/hooks/useProductsList.ts

```diff
@@ -0,0 +1,94 @@
+import {
```

Comment:
I think you're missing some documentation up here 🙂 
```
/**
 * Hook to provide ...
 *
 * @module useProductsList
 * @category Products
 * @subcategory Hooks
 */
 ```

Reply:
I totally forgot about the extra JsDoc we add here 🙈 fixed now!